src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -426,52 +426,57 @@ void compare(const ov::Tensor& expected,
     auto max_type_actual = std::numeric_limits<ActualT>::max();
     auto min_type_expected = std::numeric_limits<ExpectedT>::min();
     auto min_type_actual = std::numeric_limits<ActualT>::min();
-    Error abs_error(abs_threshold), rel_error(rel_threshold);
-    for (size_t i = 0; i < shape_size_cnt; ++i) {
+    double abs_value = 0.0;
```

Comment:
`Error` structure contains logic based on eps. Could you please revert `double` changes? Error structure accumulate the general error. In other case, please explain details why you decided to change